config/routes.rb

```diff
@@ -62,9 +63,9 @@
   put '/phone_confirmation' => 'users/phone_confirmation#confirm'
   get '/profile' => 'profile#index'
   get '/splash' => 'home#index'
-  get '/users/totp' => 'users/totp_setup#new'
-  delete '/users/totp' => 'users/totp_setup#disable', as: :disable_totp
-  patch '/users/totp' => 'users/totp_setup#confirm', as: :confirm_totp
+  get '/totp_setup' => 'users/totp_setup#new'
+  delete '/totp_setup' => 'users/totp_setup#disable', as: :disable_totp
+  patch '/totp_setup' => 'users/totp_setup#confirm'
```

Comment:
This is explicitly for TOTP apps (e.g. Google Authenticator).  So perhaps `auth_app_setup` or `authenticator_setup` or `app_setup`?